pkg/kubelet/nodestatus/setters_test.go

```diff
@@ -896,9 +897,11 @@ func TestReadyCondition(t *testing.T) {
 		cmStatus                 cm.Status
 		expectConditions         []v1.NodeCondition
 		expectEvents             []testEvent
+		isInitialized            bool
```

Comment:
Move this above the "expectation".
s/isInitialized/isRuntimeInitialized

Reply:
done